### PR TITLE
IR3 compat

### PR DESCRIPTION
**Pull request by 0n0w1c**

### CODE REVIEW

**Comment by 0n0w1c** [comment]
Two noteworthy changes, the first is plutonium-fuel is removed, it has no use in IR3, it modifies the vehicle fuels. It would of course be possible to make plutonium-fuel usable and possibly desired by some. There is no comparable energy-rich fuel in IR3. It might be worth considering adding a startup setting and adjusting the fuel category.

The second is the advanced-nuclear-fuel-reprocessing-with-barrelling technology is removed because IR3 introduces barreling machines.

**Comment by JohnTheCoolingFan** [comment]
Ok, thanks for the changes and additional notes. The auto-barrelling recipe removal would also need to be addressed in #77, I'll handle that.

File: data-updates.lua [hunk 1]
```diff
@@ -8,15 +8,7 @@ if mods['bobplates'] then
...
-if mods['IndustrialRevolution'] then
-    data.raw['ammo']['plutonium-rounds-magazine'].subgroup = 'ir2-ammo'
-    data.raw['item']['plutonium-238'].subgroup = 'pure'
-    data.raw['item']['plutonium-239'].subgroup = 'pure'
-    data.raw['item']['plutonium-fuel'].subgroup = 'ir2-fuels'
-    data.raw['item']['MOX-fuel'].order = 'ib'
-    data.raw['item']['plutonium-fuel'].order = 'fb'
-    data.raw['item']['used-up-MOX-fuel'].order = 'r[used-up-uranium-fuel-cell]-b'
-
+if mods['IndustrialRevolution3'] then
```
Comment (JohnTheCoolingFan) [inline data-updates.lua:11]: Typo: IndustrialRevolut**i**on3

File: info.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
         "? bobplates",
         "? bobwarfare",
         "? AtomicRocketScreenBurn",
-        "? IndustrialRevolution >= 2.0.0",
+        "? IndustrialRevolution >= 3.1.5",
```
Comment (JohnTheCoolingFan) [inline info.json:17]: Everywhere else IR3 is referenced as IndustrialRevolution3, here it is just IndustrialRevolution.

File: prototypes/entity/projectiles.lua [hunk 3]
```diff
@@ -254,7 +254,7 @@ if mods['bobwarfare'] then
     plutonium_atomic_artillery_projectile.action.action_delivery.target_effects[13].action.radius = 50
     table.insert(projectiles, plutonium_atomic_artillery_projectile)
 
-elseif mods['IndustrialRevolution'] then
+elseif mods['IndustrialRevoluton3'] then
```
Comment (JohnTheCoolingFan) [inline prototypes/entity/projectiles.lua:257]: Typo as well

**Review by JohnTheCoolingFan — CHANGES_REQUESTED** [review]

**Comment by 0n0w1c** [comment]
Updated info.json, sorry about that.

File: prototypes/entity/projectiles.lua (hunk 3 above)
Comment (0n0w1c) [reply prototypes/entity/projectiles.lua:257]: Updated info.json, sorry about that.

File: info.json (hunk 2 above)
Comment (0n0w1c) [reply info.json]: Updated info.json, the typo was there. IndustrialRevolution3 is the correct mod name.

File: prototypes/entity/projectiles.lua (hunk 3 above)
Comment (JohnTheCoolingFan) [reply prototypes/entity/projectiles.lua:257]: This change still has the typo

**Pull request merged**